SUPPORT.md

```diff
@@ -24,7 +26,10 @@
   - [Amazon Web Services (AWS)](#amazon-web-services-aws)
   - [MongoDB](#mongodb)
   - [Material UI](#material-ui)
+  - [Styled-Components](#styled-components)
```

Comment:
`styled-components` I am not sure if someone used it. It is a separate package and it is not needed I think. Material UI provides an API called `styled` and I already used it in `Observations.jsx` and is the same

Reply:
Fixed in a918a51 :+1: 

The original list came from an older document in the repo, so some of the info may be outdated/incorrect, thanks for catching that!